packages/docusaurus/src/client/App.js

```diff
@@ -8,12 +8,15 @@
 import React from 'react';
 import {renderRoutes} from 'react-router-config';
 
-import Head from '@docusaurus/Head';
 import routes from '@generated/routes';
 import siteConfig from '@generated/docusaurus.config';
+
+import Head from '@docusaurus/Head';
 import DocusaurusContext from '@docusaurus/context';
 import PendingNavigation from './PendingNavigation';
 
+require('@generated/client-modules');
```

Comment:
why is it require

Reply:
Oh yeah I can make it `import`. But it's really the same now. Webpack turns all of them into its own require.

Reply:
lets change it for consistency

Reply:
easier to migrate to TS later on